distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -47,8 +47,9 @@ type Column
                  Column.from_vector "My Column" [1, 2, 3, 4, 5]
     from_vector : Text -> Vector -> Column
     from_vector name items =
+        expected_storage_type = Nothing
         Illegal_Argument.handle_java_exception <|
-            Column.Value (Java_Column.fromItems name items)
+            Column.Value (Java_Column.fromItemsConvertPolyglot name items expected_storage_type)
```

Comment:
Let's not rename `fromItems` - it is our default and should remain so as a safer operation.

Reply:
Ok